```
D:\workspace\_work\1\s\.dotnet\sdk\6.0.100-rc.1.21411.28\Microsoft.Common.CurrentVersion.targets(4606,5): error MSB3026: Could not copy "D:\workspace\_work\1\s\artifacts\obj\System.Security.Principal.Windows\net6.0-windows-Release\System.Security.Principal.Windows.dll" to "D:\workspace\_work\1\s\artifacts\bin\System.Security.Principal.Windows\net6.0-windows-Release\System.Security.Principal.Windows.dll". Beginning retry 1 in 1000ms. The requested operation cannot be performed on a file with a user-mapped section open. : 'D:\workspace\_work\1\s\artifacts\bin\System.Security.Principal.Windows\net6.0-windows-Release\System.Security.Principal.Windows.dll'  [D:\workspace\_work\1\s\src\libraries\System.Security.Principal.Windows\src\System.Security.Principal.Windows.csproj]
```
Should we try to use `handle` from sysinternals to try to find out what causes this problem?

--- a/src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.Llvm.cs
+++ b/src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.Llvm.cs
@@ -59,8 +59,7 @@ namespace Internal.JitInterface
                 return;
             }
 
-            var frozenStringNode = node as FrozenStringNode;
-            if (frozenStringNode != null)
+            if (node is FrozenStringNode frozenStringNode)
             {
                 _this.AddOrReturnGlobalSymbol(frozenStringNode, _this._compilation.NameMangler);
             }

